src/histogram.rs

```diff
@@ -279,6 +288,7 @@ impl MetricVecBuilder for HistogramVecBuilder {
 pub type HistogramVec = MetricVec<HistogramVecBuilder>;
 
 impl HistogramVec {
+    /// `new` creates a HistogramVec with options `opts` and label names `label_names`.
```

Comment:
/// `new` creates a HistogramVec with the `opts` options and the `label_names` label names.